cmd/broker/main.go

```diff
@@ -23,12 +24,20 @@ func main() {
 
 	configFilepath := os.Args[1]
 	servicePath := os.Args[2]
+
 	state, err := config.NewState(configFilepath, servicePath)
 	if err != nil {
 		logger.Fatal("main", err)
 	}
 
-	broker := broker.NewBlockheadBroker(state)
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		logger.Fatal("docker-client", err)
+	}
+
+	manager := broker.NewDockerContainerManager(logger, cli)
```

Comment:
not to get too far ahead of ourselves, but should we make this a flag `--runtime=docker` so that we know where it is when kubernetes enters the picture.

Reply:
sorry, a config file entry.

the other option is we try each auto-env inspection in an order.

Reply:
i think doing it in config is reasonable